build/ci.go

```diff
@@ -463,3 +466,32 @@ func stageDebianSource(tmpdir string, meta debMetadata) (pkgdir string) {
 
 	return pkgdir
 }
+
+// Cross compilation
+
+func doXgo(cmdline []string) {
+	// Make sure xgo is available for cross compilation
+	gogetxgo := goTool("get", "github.com/karalabe/xgo")
+	build.MustRun(gogetxgo)
+
+	// Execute the actual cross compilation
+	pkg := cmdline[len(cmdline)-1]
+	args := append(cmdline[:len(cmdline)-1], makeBuildFlags("")...)
```

Comment:
Would be nice to have the commit hash in cross compiled binaries as well. I guess we can do that later if/when we import xgo rather than call it as a command.